app/v2/partials/directives/proposals-ideas-searchbox.html

```diff
@@ -124,6 +124,9 @@ <h4 class="widget__label">
       <li class="sarch_mode__popularity" ng-click="setSorting('popularity')" ng-class="{active: sortingIs('popularity')}">
         <a href="">{{'Popularity' | translate}}</a>
       </li>
+      <li class="sarch_mode__popularity" ng-click="setSorting('most_commented')" ng-class="{active: sortingIs('most_commented')}">
```

Comment:
Let's use the following rule to choose the right "most_commented" option: 
- [ ] If `isAnonymous === TRUE`, use `most_commented_public`
- [ ] If `isAnonymous === FALSE`, then
     - [ ] If campaign configs includes `appcivist.campaign.disable-public-site` and it is equal to `FALSE`, then use `most_commented`
     - [ ] If the previous is not true, then use `most_commented_members`